content/docs/04-clusters/03-edge/11-deploy-cluster.md

```diff
@@ -0,0 +1,856 @@
+
+
+
+Clicking on the exposed NodePort will take you to the Hello Universe application. 
+<br />
+
+<WarningBox>
+
+We recommend waiting to click on the service URL, as it takes one to three minutes for DNS to properly resolve the public NodePort URL. This prevents the browser from caching an unresolved DNS request.
+
+</WarningBox>
+
+
+![Screenshot of successfully accessing the Hello Universe application.](/tutorials/edge-native/clusters_edge_deploy-cluster_hello-universe.png "#width=70%")
+
+
+You can also review at real-time metrics in Palette, such as CPU and memory consumption, in the cluster's **Overview** tab in Palette. 
+
+![Screenshot of cluster metrics.](/tutorials/edge-native/clusters_edge_deploy-cluster_metrics.png)
+
+You have successfully provisioned an Edge cluster and deployed the Hello Universe application on it. 
+
+<br />
+
+# Cleanup
+The following steps will guide you in cleaning up your environment, including the cluster, cluster profile, and Edge hosts. 
+<br />
+
+##  Delete the Cluster and Profile
+To delete the cluster, view the cluster details page. Click on the **Settings** button to expand the **drop-down Menu**, and select the **Delete Cluster** option, as shown in the screenshot below
+
+
+![Screenshot of deleting a cluster.](/tutorials/edge-native/clusters_edge_deploy-cluster_delete-cluster.png)
+
+
+Palette will prompt you to enter the cluster name and confirm the delete action. Type the cluster name to proceed with the delete step. The cluster status will turn to **Deleting**. Deletion takes up to 10 minutes.
+
+
+After you delete the cluster, click **Profiles** on the left **Main Menu**, and select the profile to delete. Choose the **Delete** option in the **three-dot Menu**, as shown in the screenshot below.
+
+
+![Screenshot of deleting a cluster profile.](/tutorials/edge-native/clusters_edge_deploy-cluster_delete-profile.png)
+
+
+Please wait until Palette deletes the resources successfully. 
+
+##  Delete Edge Hosts
+Switch back to the **CanvOS** directory in the Linux machine containing the **.goenv** file, and use the following command to delete the Edge hosts. 
+<br />
+
+```bash
+docker run -it --rm --env-file .goenv tutorials sh -c "cd edge-native/vmware/clone_vm_template/ && ./delete-edge-host.sh"
+```
+
+##  Delete Edge Artifacts
+If you further want to delete Edge artifacts from your Linux machine, delete the Edge installer ISO image and its checksum by executing the following commands from the **CanvOS/** directory.
+<br />
+
+```bash
+rm build/palette-edge-installer.iso
+rm build/palette-edge-installer.iso.sha256
+```
+
+Issue the following command to list all images in your current development environment.
+<br />
+
+```bash
+docker images
+```
+
+Note the provider image name and tags, and use the following command syntax to remove all provider images. Similarly, repeat this command for deleting the Earthly image. 
+
+<br />
+
+```bash
+docker image rm -f [image repository name]:[tag]
+```
```

Comment:
Show the the commands to clean the images used in this tutorial. You did it for the docker push command, so let's also do it here.